File: Demo/FrmMain.vb
Public Class FrmMain
    Private Sub BtnDemo1_Click(sender As Object, e As EventArgs) Handles BtnDemo1.Click
        Dim f As FrmDemo1 = New FrmDemo1()
        f.ShowDialog()
    End Sub

    Private Sub BtnDemo2_Click(sender As Object, e As EventArgs) Handles BtnDemo2.Click
        Dim f As FrmDemo2 = New FrmDemo2()
        f.ShowDialog()
    End Sub

    Private Sub BtnDemo3_Click(sender As Object, e As EventArgs) Handles BtnDemo3.Click
        Dim f As FrmDemo3 = New FrmDemo3()
        f.ShowDialog()
    End Sub
End Class

File: DemoCS/FrmMain.cs
using System;
using System.Windows.Forms;

namespace DemoCS
{
    public partial class FrmMain : Form
    {
        public FrmMain()
        {
            InitializeComponent();
        }

        private void BtnDemo1_Click(object sender, EventArgs e)
        {
            FrmDemo1 f = new FrmDemo1();
            f.ShowDialog();
        }

        private void BtnDemo2_Click(object sender, EventArgs e)
        {
            FrmDemo2 f = new FrmDemo2();
            f.ShowDialog();
        }

        private void BtnDemo3_Click(object sender, EventArgs e)
        {
            FrmDemo3 f = new FrmDemo3();
            f.ShowDialog();
        }
    }
}
